packages/ui/src/stories/Button/button.style.ts

```diff
@@ -0,0 +1,87 @@
+import styled from "styled-components";
+
+const Colors = {
+  primary: "#0071f3",
+  secondary: "#ACB1B6",
+  active: "#0366d6",
+  activeLight: "#5da2f0",
+  textPrimary: "#031323",
+  divider: "#D9DCDE",
+  blackPearl: "#59636e",
+  primaryDim: "#FBF7FE",
+
+  destructive: "#E63644",
+  affirmative: "#189F59",
+  warning: "#D97508",
+  white: "#FFFFFF",
+  destructiveDim: "#FFE1E1",
+  affirmativeDim: "#E8F5E5",
+  warningDim: "#FFF6D1",
+  textInactive: "#BFBFBF",
+  background: "#F2F3F4",
+};
```

Comment:
As you said is better if we export this.

Reply:
Done.